### DISCUSSION TITLE
Calendar heatmaps

### DISCUSSION THREAD

**Discussion by gautam-kumar71** [monkeytypegame/monkeytype#4350] [Ideas] [answered]
I'm a dedicated user of Monkey-type and would like to suggest adding calendar heatmaps similar to Codeforces or LeetCode. Calendar heatmaps would visually represent our typing streaks, motivating consistent practice and allowing us to track progress over time. This feature would greatly enhance the user experience and engagement. Thank you for considering this valuable addition.
![calendar_heatmap](https://github.com/monkeytypegame/monkeytype/assets/81165310/83dbe711-0562-4558-b73c-b0f00b192814)

**Comment by Miodec** [comment, marked as answer]
The third graph on the account page shows your activity over time
<img width="1179" alt="image" src="https://github.com/monkeytypegame/monkeytype/assets/13181393/10af2abc-ded2-4cae-b14d-50600bb83456">
